aws-organizations-resourcepolicy/src/test/java/software/amazon/organizations/resourcepolicy/CreateHandlerTest.java

```diff
@@ -86,10 +87,11 @@ public void handleRequest_NoTags_SimpleSuccess() {
 
     @Test
     public void handleRequest_WithTags_SimpleSuccess() {
-        final ResourceModel model = generateInitialResourceModel(false, TEST_RESOURCEPOLICY_CONTENT);
```

Comment:
This and few others had incorrectly set the `hasTags` to false, have updated them to true